### PR TITLE
Every task done

### PR DESCRIPTION
**Pull request by EPPDL**
Everything completed, waiting for your answer

### CODE REVIEW

File: app1/src/App.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
+import * as React from 'react';
 import logo from './logo.svg';
 import './App.css';
 import MainCard from './components/cards/MainCard';
 import InputField from './components/fields/InputField';
 
 function App() {
+
+  const [newState, setNewState] = React.useState('Eduardo');
```
Comment (rspineda) [inline app1/src/App.js]: state name should be representative like 'displayedName'.

File: app1/src/App.js [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+   const getPersonName = (enteredPersonName) => {
+    const personName = enteredPersonName;
+    console.log(personName);
```
Comment (rspineda) [inline app1/src/App.js:12]: remove console.log()

File: app1/src/components/cards/MainCard.js [hunk 3]
```diff
@@ -5,12 +5,15 @@ import CardMedia from '@mui/material/CardMedia';
...
 
-export default function MainCard() {
-    const [nombre, setNombre] = React.useState('Santiago Loberto');
-    const handler = () => {
-        setTimeout(()=> setNombre('Gustavo Dazarin') ,2000);
-    }
+export default function MainCard(props) {
 
+  const [nombre, setNombre] = React.useState('Hola');
+    const handle = () => {
```
Comment (rspineda) [inline app1/src/components/cards/MainCard.js:11]: method name should be more representative

File: app1/src/components/cards/MainCard.js [hunk 4]
```diff
@@ -5,12 +5,15 @@ import CardMedia from '@mui/material/CardMedia';
...
+        setNombre(props.newName);
+        console.log(props.newName);
```
Comment (rspineda) [inline app1/src/components/cards/MainCard.js]: remove console.log()

File: app1/src/components/fields/InputField.js [hunk 5]
```diff
@@ -32,37 +33,45 @@ const names = [
...
       target: { value },
     } = event;
     setPersonName(
       // On autofill we get a the stringified value.
       typeof value === 'string' ? value.split(',') : value,
     );
-  };
+  };*/
+
+    const handleSimpleChange = (event) => {
```
Comment (rspineda) [inline app1/src/components/fields/InputField.js]: method name should be more representative like 'SelectNameHandler'

**Comment by EPPDL** [comment]
I already fixed the code

**Pull request merged**